src/css/style.css

```diff
@@ -227,6 +227,12 @@ button {
     font-size: inherit;
     grid-template-columns: 150px auto 15%;
   }
+    
+  img {
+    display: flex;
+    justify-content: space-between;
+  }
+
```

Comment:
Is there any way you can adjust the size of the logo in the top left when the screen size is mobile?